src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotChangeEventSource.java

```diff
@@ -604,22 +604,8 @@ private OpId getTransactionStartLsn() throws SQLException {
     protected void complete(SnapshotContext<YBPartition, YugabyteDBOffsetContext> snapshotContext) {
         snapshotter.snapshotCompleted();
 
-        // Close the YBClient instances now
-        if (asyncClient != null) {
-          try {
-            asyncClient.close();
-          } catch (Exception e) {
-            LOGGER.warn("Error while closing async YBClient", e);
-          }
-        }
-
-        if (syncClient != null) {
-          try {
-            syncClient.close();
-          } catch (Exception e) {
-            LOGGER.warn("Error while closing YBClient", e);
-          }
-        }
+        // Todo Vaibhav: Close the YBClient instances now
+        // See if it can be closed anywhere else for the snapshotting tasks.
```

Comment:
I have removed the closure of clients because it seems the `execute()` function is being called per partition and once it finishes, it closes the YBClient instance, which are then rendered unusable later on for the same task so it started giving exceptions that the yb-client object has already been closed.

Currently I am relying on the garbage-collection mechanism that it will collect this open object, but I will soon raise another PR to fix this specific issue.

Reply:
Can you also create a issue?

Reply:
Created issue #80